cherryontech/comet-cruisers-spring2024
# Update/delete confirmation

## Jira Story Card / Issue Link
Sprint 3 Security/Confirmation Check before deleting Entry

## Type of change in this PR:
- if statement using confirm function
## Testing instructions:
make a sample journal entry. click delete. If you click cancel, it should keep the entry. If you click ok, the entry should be deleted.
